dev_container

```diff
@@ -540,10 +540,12 @@ launch() {
     if [ -d  "$local_sdk_root" ] ; then
         local_sdk_opt+=" -v $local_sdk_root:/workspace/holoscan-sdk"
          #   Define paths needed by the python applications
-        local_sdk_opt+=" -e PYTHONPATH=/workspace/holoscan-sdk/build/python/lib"
+        local_sdk_opt+=" -e PYTHONPATH=/workspace/holoscan-sdk/build/python/lib:/workspace/holohub/benchmarks/holoscan_flow_benchmarking"
```

Comment:
Would be nice to maintain the path in only one place, with something like:

```bash
local container_pythonpath="/workspace/holohub/benchmarks/holoscan_flow_benchmarking"
if [ ... ]; then
  container_pythonpath="/workspace/holoscan-sdk/build/python/lib:${container_pythonpath}"
else
  container_pythonpath="/opt/nvidia/holoscan/python/lib:${container_pythonpath}"
fi
local_sdk_opt+=" -e PYTHONPATH=${container_pythonpath}
```

Reply:
thanks @tbirdso , updated